### PR TITLE
Added possibility to define Paysera application routes via environment variables

### PR DESCRIPTION
**Pull request by dankovalenko-paysera**
Instead of using Paysera routes via constants, the possibility to define routes in environment variables is added. It may be useful for  debugging

Added configuration object, which composes routes and other possible configs (`projectId`, `password` for now), instead of using arrayed configurations. So configurations are typed now

### CODE REVIEW

**Comment by aliaksandr-falei** [comment]
Please, fix the typo in "Tests" section of Readme: "bash run**t**_tests.sh "

**Comment by dankovalenko-paysera** [comment]
@aliaksandr-falei fixed

File: tests/WebToPay/Config/EnvReaderTest.php [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+     * @dataProvider getAsArrayData
+     */
+    public function testGetAsString(string $key, ?string $expected, ?string $actual): void
+    {
+        $this->assertEquals($expected, $actual);
+    }
+
+    public function getAsArrayData(): iterable
+    {
+        $envReader = new WebToPay_EnvReader();
```
Comment (aliaksandr-falei) [inline tests/WebToPay/Config/EnvReaderTest.php:19]: A bit strange to see SUT here, in data provider. Usually, data provider is just to provide various inputs to the test. And SUT can reside in setUp or test itself.

Also, I'd put manipulation with $_ENV and printenv to the setUp / tearDown.

File: src/WebToPay/Config/Config.php [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+
+    /**
+     * @throws Exception
+     */
+    protected function initEnvVar(string $varName, string $targetProperty): void
+    {
+        $envValue = getenv($varName);
+
+        if (empty($envValue)) {
+            $envValue = static::DEFAULT_VALUES[$targetProperty];
```
[the remark this answers is not in the document]
Comment (aliaksandr-falei) [reply src/WebToPay/Config/Config.php]: ok

**Review by aliaksandr-falei — APPROVED** [review]

**Review by aliaksandr-falei — APPROVED** [review]

File: tests/WebToPay/Config/EnvReaderTest.php (hunk 1 above)
Comment (dankovalenko-paysera) [reply tests/WebToPay/Config/EnvReaderTest.php]: Done, changed the test

**Review by aliaksandr-falei — APPROVED** [review]

**Review by aliaksandr-falei — APPROVED** [review]

**Review by aliaksandr-falei — APPROVED** [review]

**Pull request merged**